config/sara/sara.yml

```diff
@@ -7,8 +7,24 @@ base_path: ''
 # How many days an enrolled monitoree should be monitored
 monitoring_period_days: <%= ENV["MONITORING_PERIOD_DAYS"] || 10 %>
 
-# How many days an enrolled case should be monitored until recovery
-recovery_period_days: <%= ENV["RECOVERY_PERIOD_DAYS"] || 5 %>
+# The following config values control certain variables in the recovery definitions for the Isolation Workflow,
+# including whether they're enabled. See the actual recovery definitions in the Patient model for a thorough
+# understanding of how these values are used.
+# 
+# Corresponding scope in patient model: isolation_asymp_non_test_based
+enable_asymp_non_test_based: <%= ENV["ENABLE_ASYMP_NON_TEST_BASED"] || true %>
+asymp_non_test_based_recovery_period_days: <%= ENV["ASYMP_NON_TEST_BASED_RECOVERY_PERIOD_DAYS"] || 5 %>
+
+# Corresponding scope in patient model: isolation_symp_non_test_based
+enable_symp_non_test_based: <%= ENV["ENABLE_SYMP_NON_TEST_BASED"] || true %>
+symp_non_test_based_recovery_period_days: <%= ENV["SYMP_NON_TEST_BASED_RECOVERY_PERIOD_DAYS"] || 5 %>
+symp_non_test_based_hours_since_fever: <%= ENV["SYMP_NON_TEST_BASED_HOURS_SINCE_FEVER"] || 24 %>
```

Comment:
Could you add a comment here that this value (and the one on line 26) also apply for the time period since use of a fever reducer was last reported?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1238/commits/00bf9c196aca215979d31cb39a6e81a750e4088f